cfme/tests/test_db_migrate_manual.py

```diff
@@ -288,19 +288,25 @@ def test_update_webui_custom_css():
         1553841
 
     Test css customization"s function correctly after webui update.
+    Old bugzilla, css customisation is not possible to be done in production,
+    but custom.css file can be changed
 
     Polarion:
         assignee: jhenner
         casecomponent: Appliance
         caseimportance: medium
         initialEstimate: 1/6h
-        setup: provision appliance
-               add custom css file
-               add repo file to /etc/yum.repos.d/
-               add RHSM settings
-               update through webui
-               check webui is available
-               check customization"s still work
+
+        setup:  1. provision appliance
```

Comment:
As far as I can tell, this should be part of the `testSteps` and not the `setup`? If so, please move to `testSteps` and give corresponding `expectedResults`